### ISSUE TITLE
loss = 0.0000

### ISSUE THREAD

**Issue by benlee73** [ZJULearning/pixel_link#168]
![image](https://user-images.githubusercontent.com/55429965/109754077-c293ce00-7c26-11eb-998a-d094cbb9a7f3.png)

I don't know why loss is zero.
The training worked well with the icdar2015 dataset, but when I used my other dataset image and ground truth, I got this result.
The format of ground truth is the same as icdar2015.
How can I fix the problem?

For reference, my image size is about 601x792.
And i'm not change another config or something.
